.github/workflows/codegen.yml

```diff
@@ -12,12 +12,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        platform: ["android"] # "ios" will be added after codegen is fixed
+        platform: ["android", "ios"]
         include:
           - platform: android
             command: |
               cd sample-new-architecture/android
               ./gradlew generateCodegenArtifactsFromSchema
+          - platform: ios
+            command: |
+              cd sample-new-architecture
```

Comment:
`l`: out of curiosity - is there a reason why this is not under `sample-new-architecture/ios`?

Reply:
No, it could be and then the `-path` to the app would be `..`.